globals/global_settings.gd

```diff
@@ -6,13 +6,19 @@ var MuteEmotes = false
 const ClientVersionString : String = "240505.1500" # YYMMDD.HHMM
 
 const CharacterBanlist = ['carmine']
+const DefaultStartingTimer = 15
+const DefaultStartingBestOf = 1
+const DefaultEnforceTimer = true
 
 # Persistent Settings
 var BGMEnabled = true
 var DefaultPlayerName = ""
 var GameSoundsEnabled = true
 var PlayerCharacter = ""
 var CombatLogSettings = {}
+var CustomStartingTimer = DefaultStartingTimer
+var CustomStartingBestOf = DefaultStartingBestOf
+var EnforceTimer = DefaultEnforceTimer
```

Comment:
Do you want any of these to persist? If so, you'll have to add more code like for the other settings.
If not, I don't want these under the # Persistent Settings comment.

Reply:
they should now be persistent